README.md

```diff
@@ -51,4 +49,4 @@ Any contributions made under this project will be governed by the [Apache Licens
 ## 🗣️ Code of Conduct
 
 This project adheres to the [American Express Community Guidelines](./CODE_OF_CONDUCT.md).
-By participating, you are expected to honor these guidelines.
\ No newline at end of file
+By participating, you are expected to honor these guidelines.
```

Comment:
Why did this change?

Reply:
I have no idea. I don't remember touching that. I will add that back.